manifests/mcollective.pp

```diff
@@ -1,7 +1,7 @@
 define clamps::mcollective (
-  $user = $title,
-  $amqpass = 'password',
-  $amqserver = '$::server',
+  $user       = $title,
+  $amqpass    = 'password',
+  $amqservers = [$::servername],
```

Comment:
Thats a good question. I just took the value that was there and converted it to an array, and its the default we use in the module. 

@doug-rosser Did you run into any issues around this when doing your testing since it was all in ec2?